_posts/2022-05-18-animated-scroll-effects.md

```diff
@@ -0,0 +1,136 @@
+---
+layout: post
+title: "Animated Scroll Effects"
+date: 2022-05-18 12:00:00
+tags:
+- CSS
+- SCSS
+- JavaScript
+- UI/UX
+- frontend
+categories:
+- development
+twitter_text:
+authors: Ryan Trimble
+image: https://images.unsplash.com/photo-1627163439134-7a8c47e08208?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=2532&q=80
+image_url: https://unsplash.com/photos/R5A_YlcSJwA
+image_credit: Shubham Dhage
+---
+
+I’m a huge fan of animation on the web, especially when it comes to user interaction. A simple way to include such a thing on webpages is by triggering some sort of effect to occur when scrolling.
```

Comment:
I can't put my finger on it but the second sentence reads weird to me. I know you don't want to say animation again but `such a thing` is making me feel like I should be drinking my tea with my pinky out while reading this. I also don't know that you need `to occur when`. could it just be:
```suggestion
I’m a huge fan of animation on the web, especially when it comes to user interaction. A simple way to include such a thing on webpages is by triggering some sort of effect when scrolling.
```